requirements.txt

```diff
@@ -3,4 +3,5 @@ flake8
 python-novaclient==7.0.0
 PyYAML
 texttable
-paramiko==2.1.2
\ No newline at end of file
+# temporarily disabled
```

Comment:
let's separate requirements.txt file to requirements.txt and benchmark/openstack_requirements.txt - to be more agile. I hope that user in this case may choose the way what she/he needs and install desired stuff, and most important part that repo will have all requirements in place. @legan4ik what do you think?